pkg/sdk/testint/procedures_integration_test.go

```diff
@@ -0,0 +1,541 @@
+package testint
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/collections"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk/internal/random"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_CreateProcedures(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	_, warehouseCleanup := createWarehouse(t, client)
```

Comment:
there are common resources prepared (schema, db and warehouse specifically). Is there any valid reason for creating the new ones here?

Reply:
replaced